DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/Index.cshtml

```diff
@@ -0,0 +1,114 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.Common
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates
+@using Microsoft.Extensions.Configuration
+@model ViewDelegateViewModel
+
+<link rel="stylesheet" href="@Url.Content("~/css/trackingSystem/viewDelegate.css")" asp-append-version="true">
+
+@{
+  ViewData["Title"] = "View delegate";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+
+  var activeTagCss = Model.DelegateInfo.IsActive ? "nhsuk-tag nhsuk-tag--green" : "nhsuk-tag nhsuk-tag--red";
+  var passwordTagCss = Model.DelegateInfo.IsPasswordSet ? "nhsuk-tag nhsuk-tag--green" : "nhsuk-tag nhsuk-tag--red";
+
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}
+
+@section NavBreadcrumbs {
+  <partial name="~/Views/TrackingSystem/Delegates/Shared/_AllDelegatesBreadcrumbs.cshtml" />
+}
+
+<div class="nhsuk-grid-row">
+  <div class="nhsuk-grid-column-full">
+    <div class="nhsuk-grid-row">
+      <div class="nhsuk-grid-column-two-thirds">
+        <h1 class="nhsuk-heading-xl nhsuk-u-margin-bottom-3">@Model.DelegateInfo.Name</h1>
+      </div>
+      @if (Model.DelegateInfo.IsActive) {
+        <div class="nhsuk-grid-column-one-third view-delegate-top-button-group">
+          <a class="nhsuk-button nhsuk-button--secondary view-delegate-button " role="button">
+            Send Welcome Email
+          </a>
+        </div>
+      }
+    </div>
+
+    <div class="nhsuk-u-margin-bottom-5">
+      <strong class="@activeTagCss">@(Model.ActiveTagName)</strong>
+      <strong class="nhsuk-tag nhsuk-tag--grey">@(Model.RegStatusTagName)</strong>
+      <strong class="@passwordTagCss">@(Model.PasswordTagName)</strong>
+      @if (Model.DelegateInfo.IsAdmin) {
+        <strong class="nhsuk-tag nhsuk-tag--grey">Admin</strong>
+      }
+    </div>
+
+    <h2 class="nhsuk-heading-l nhsuk-u-margin-bottom-2">Details</h2>

```

Comment:
If this set of details is very similar in display to the All Delegates one, is it worth combining them into a partial to use in both places?

Reply:
I did actually start off with that, but after discussing with Stella we decided to scrap that as it could later on become a nuisance to modify the styling or fields in the two different places.